### DISCUSSION TITLE
can't download files from my Backblaze B2 bucket

### DISCUSSION THREAD

**Discussion by nnachtegael** [iterate-ch/cyberduck#15324] [Q&A]
I'm new with Cyberduck and Backblaze. 
I successfully uploaded files from my windows 11 system to a Backblaze b2 bucket with Cyberduck version 8.7.0(40629).
I can succesfull download my files back to my system using the Backblaze download commands from my Backblaze account but I can't download those files back to my system using Cyberduck.
Is this a kwnown issue? Is there a solution?

**Comment by dkocher** [comment]
This is not a known issue. Please let us know what error you get. Please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level) in _Preferences → Connection_  and attach the log file.

**Comment by nnachtegael** [comment]
[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/13250237/cyberduck.log)

**Comment by nnachtegael** [comment]
I have a lot of connection refused in my log file. May be a certificate issue like mentionned [here](https://help.backblaze.com/hc/en-us/articles/4408304684699-Cannot-Connect-to-Backblaze-due-to-Security-Certificate-Win)

**Reply by dkocher** [reply]
The `Connection refused: connect` errors are not related to issues with TLS but general connectivity issues to B2 servers not responding.

**Comment by dkocher** [comment]
Try to increase the timeout [^1] in _Preferences → Connection → Timeouts_.

[^1]: https://docs.cyberduck.io/cyberduck/connection/#i-get-an-error-io-error-read-timed-out

**Comment by nnachtegael** [comment]
my preferences are already 
timeout 30 sec
wait 10 sec
retries 20

**Comment by nnachtegael** [comment]
I saw several  'org.apache.http.conn.HttpHostConnectException' in the log 
[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/13251620/cyberduck.log)

**Comment by nnachtegael** [comment]
More info about connection in windows 11
[Connection-refused.txt](https://github.com/iterate-ch/cyberduck/files/13251924/Connection-refused.txt)

**Comment by sedlund** [comment]
Backblaze has been offering S3 compatibility for a while now.  You might have better luck configuring cyberduck to connect using S3.

**Reply by nnachtegael** [reply]
OK thank you, it's working using S3
